consensus/state_processing/src/per_block_processing/signature_sets.rs

```diff
@@ -387,12 +387,23 @@ where
     let exit = &signed_exit.message;
     let proposer_index = exit.validator_index as usize;
 
-    let domain = spec.get_domain(
-        exit.epoch,
-        Domain::VoluntaryExit,
-        &state.fork(),
-        state.genesis_validators_root(),
-    );
+    let domain = match state {
+        BeaconState::Base(_)
+        | BeaconState::Altair(_)
+        | BeaconState::Merge(_)
+        | BeaconState::Capella(_) => spec.get_domain(
```

Comment:
Not in this case because [`get_domain`](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_domain) is a spec-defined function & changes of this type are done via bypassing this function to instead go straight to [`compute_domain`](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#compute_domain). Compare `process_voluntary_exit` in:
 * [`phase0`](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#voluntary-exits)
  * [`deneb`](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/beacon-chain.md#modified-process_voluntary_exit)

Reply:
Right! Makes sense